test/language/expressions/tagged-template/cache-differing-expressions-eval.js

```diff
@@ -4,10 +4,8 @@
 es6id: 12.2.8
```

Comment:
this behavior is not valid anymore for es6, these tags should be replaced by respective `esid` tags as in `esid: sec-gettemplateobject`